Controllers/CarteiraInvestimentoController.cs

```diff
@@ -0,0 +1,141 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using PRIORI_SERVICES_API.Model;
+using PRIORI_SERVICES_API.Models;
+using PRIORI_SERVICES_API.Models.Dbos;
+using System.Data;
+
+namespace PRIORI_SERVICES_API.Controllers
```

Comment:
Não precisa escopar o arquivo inteiro, tu pode só colocar assim:

```suggestion
namespace PRIORI_SERVICES_API.Controllers;
```